CMakeLists.txt

```diff
@@ -17,19 +20,34 @@ include(AsteroidCMakeSettings)
 set(ASTEROID_MODULES_INSTALL_DIR ${CMAKE_INSTALL_DATADIR}/asteroidapp/cmake)
 
 find_package(Qt5 ${QT_MIN_VERSION} COMPONENTS DBus Qml Quick Svg REQUIRED)
-find_package(Mlite5 MODULE REQUIRED)
-find_package(Mapplauncherd_qt5 MODULE REQUIRED)
+if (WITH_ASTEROIDAPP)
+    find_package(Mlite5 MODULE REQUIRED)
```

Comment:
*Just a side-note*, it's not relevant for this PR: I no longer see the need for this mlite dependency. Probably it could be entirely removed

Reply:
I'll test that theory and do another PR later if I can successfully eliminate it.